packages/components/src/components/hds/flyout/index.ts

```diff
@@ -187,5 +188,13 @@ export default class HdsFlyout extends Component<HdsFlyoutSignature> {
         this.body.style.setProperty('overflow', this.bodyInitialOverflowValue);
       }
     }
+
+    // Return focus to a specific element (if provided)
+    if (this.args.returnFocusTo) {
+      const initiator = document.getElementById(this.args.returnFocusTo);
```

Comment:
I am using `getElementById` which means that if consumers want to use the `returnFocusTo` argument, they **_have to_** assign an `id` to the element they want to return the focus to.

The reasoning for not being less "strict" and use something like `querySelector` is that one would be tempted to use a CSS class name (which may not be unique in the page) or a data attribute (but if you can assign a data attribute, you can also assign an `id`)

Can you think of cases where they would not be able to assign an ID to the target element, or the ID is dynamically generated?

Reply:
I also prefer the approach of using an `id` here. We can have folks try it out, I'm sure they'll tell us if they can't, but I don't know of any immediate reason why they wouldn't be able to do that.

Reply:
> We can have folks try it out, I'm sure they'll tell us if they can't

That's what they are using in Cloud UI, so I _think_ it should work OK: https://github.com/hashicorp/cloud-ui/blob/main/addons/core/package/src/components/modal-dialog.ts#L236-L241